Classes/View/PdfView.php

```diff
@@ -127,7 +127,7 @@ private function replaceStrings($content) {
             }
         }
 
-        return $content;
+        return preg_replace('/href=\"(.*?)#(.*?)\"/', 'href="#$2"', $content);
```

Comment:
Won't this break links to actual external URLs? For example when `a.html` is printed and contains a link to `b.html#foo`?